URLSessionCancellation/Sources/URLSession+Cancellation.swift

```diff
@@ -33,9 +33,9 @@ extension URLSession {
 
     // MARK: - Cancel requests
 
-    func cancelAllRequests() {
+    open func cancelAllRequests() {
```

Comment:
Since your `podspec` uses the `Sources` folder in the root of the repo, why not use a reference to that file's version rather than maintaining duplicate copies?

Also, to prevent issues like this from happening again, why not create a separate project for the Framework and then a workspace to contain the framework and the example projects. That way the example can reference the framework output and you can properly test scoping of your members when accessed as an external consumer of the framework? See FormValidator-Swift as an example of doing this.